src/index.html

```diff
@@ -4,9 +4,311 @@
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Smartphones</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Gadgets</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Tablets</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Photo</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price__word">Price:</p>
+          <p class="card__price__value">$2,199</p>
+        </div>
+
+        <a href="#"
+          class="card__button"
+          data-qa="card-hover">
+          Buy
+        </a>
+      </div>
+
+      <div class="card card--location" data-qa="card">
+        <img src="images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
+
+        <h3 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars--4">
+            <div class="stars__star"></div>
```

Comment:
All of these tags could be a li tag instead of div. It would be semantically better.  

```suggestion
            <li class="stars__star"></li>
```

Further info:
[See HTML Semantic Elements ](https://www.w3schools.com/html/html5_semantic_elements.asp)
[See Use semantic tags where possible](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-content)